style.css

```diff
@@ -0,0 +1,146 @@
+html,
+body {
+  background-color: #f4f5f6;
+  min-width: 460px;
```

Comment:
- [optional ] I believe that this is making the page not work in mobile devices. It is breaking below this. Since it was not pointed out in the first review, it is optional. but it would be nice if it behaved like in the original.

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/92959234-d3b30d80-f441-11ea-8831-66930b6134f8.png)|![image](https://user-images.githubusercontent.com/61479648/92959274-e6c5dd80-f441-11ea-90bc-119c7963cb6c.png)  |